src/client/components/pages/Courses/CoursesPage.tsx

```diff
@@ -425,34 +428,45 @@ const CoursesPage: FunctionComponent = (): ReactElement => {
             onChange={toggleColumn}
           />
         </ViewModal>
-        <Button
-          variant={VARIANT.INFO}
-          forwardRef={customizeViewButtonRef}
-          onClick={() => {
-            setViewModalVisible(true);
-          }}
-        >
-          <FontAwesomeIcon
-            icon={faWrench}
+        <MenuFlex>
+          <Button
+            variant={VARIANT.INFO}
+            forwardRef={customizeViewButtonRef}
+            onClick={() => {
+              setViewModalVisible(true);
+            }}
+          >
+            <FontAwesomeIcon
+              icon={faWrench}
+            />
+            {' '}
+            Customize View
+          </Button>
+          <Checkbox
+            id="showRetiredCheckbox"
+            name="showRetiredCheckbox"
+            label="Show Retired"
+            checked={showRetired}
+            onChange={() => setShowRetired(!showRetired)}
+            labelPosition={POSITION.RIGHT}
+            hideError
           />
-          {' '}
-          Customize View
-        </Button>
-        <Dropdown
-          id="academic-year-selector"
-          name="academic-year-selector"
-          label="Academic Year"
-          isLabelVisible
-          options={academicYearOptions}
-          value={selectedAcademicYear.toString()}
-          onChange={
-            ({
-              target: { value },
-            }: ChangeEvent<HTMLSelectElement>) => {
-              setSelectedAcademicYear(parseInt(value, 10));
+          <Dropdown
+            id="academic-year-selector"
+            name="academic-year-selector"
+            label="Academic Year"
+            isLabelVisible
+            options={academicYearOptions}
+            value={selectedAcademicYear.toString()}
+            onChange={
+              ({
+                target: { value },
+              }: ChangeEvent<HTMLSelectElement>) => {
+                setSelectedAcademicYear(parseInt(value, 10));
+              }
             }
-          }
-        />
+          />
```

Comment:
Can we just swap the order of the Academic Year Selector and the Show Retired button? I think it makes a little more sense for that to be more prominent in the middle of the page

Reply:
Thanks that makes sense! I updated this in [38b5501](https://github.com/seas-computing/course-planner/pull/460/commits/38b5501c44260129d6f93cd4389ed1946995f224).